Configuration.cs

```diff
@@ -11,7 +11,6 @@ public class Configuration : IPluginConfiguration
 
         public bool alwaysOn { get; set; } = false;
         public int delta { get; set; } = 1;
-        public bool hq { get; set; } = false;
```

Comment:
Curious if this is safe to do without bumping the config version and doing something more careful. Did you old config work fine after making this change (minus the hq setting obviously?)

Reply:
Not sure if I understand the meaning of "bumping the config version"
After removing this leftover - everything was still working

Reply:
Cool. Wasn't sure if the old config settings would get read incorrectly or something.